pytorch_lightning/trainer/trainer.py

```diff
@@ -118,10 +118,13 @@ def __init__(
             resume_from_checkpoint: Optional[str] = None,
             profiler: Optional[BaseProfiler] = None,
             benchmark: bool = False,
+            deterministic: bool = False,
```

Comment:
do we need an extra parameter, what about link deterministic with seed, if any seed is set then set
```
torch.backends.cudnn.deterministic = True
```

Reply:
There might be a problem because if seed is not set I am publishing the seed that would be there(in range 0 to uint32). So you might not be interested to set exact seed for each run, but you want to have ability to restore it and reproduce the results if needed.

Reply:
deterministic and seed are not the same...

![image](https://user-images.githubusercontent.com/3640001/80865890-0a8a4300-8c5a-11ea-9b5d-1d6cc72632da.png)